bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/lib/structure/devices/impl/DeviceImpl.java

```diff
@@ -269,21 +265,41 @@ public String getHWinfo() {
 
     @Override
     public List<Short> getGroups() {
-        return new LinkedList<>(groupList);
+        LinkedList<Short> linkedList = new LinkedList<>();
+        groupList.forEach(group -> linkedList.add(group.getId()));
+        return linkedList;
+    }
+
+    /**
+     * Adds the ApplicationGroup of the given groupId to the internal list
+     * 
+     * @param groupID
+     * @return true if the groupId is a valid ApplicationGroup id, false otherwise
+     */
+    private boolean addGroupToList(Short groupID) {
+        ApplicationGroup group = ApplicationGroup.getGroup(groupID);
+        if (ApplicationGroup.UNDEFINED.equals(group)) {
+            logger.info("Unknown application group with ID '{}' found! Ignoring group", groupID);
```

Comment:
Logging to info should be used sparsely e.g. a newly started component or a user file that has been loaded. This could be warn.

Reply:
I've changed the log output to warning.